pxtpy/pydecompiler.ts

```diff
@@ -1213,8 +1219,10 @@ namespace pxt.py {
             return asExpRes(`(${expToStr(inner)})`, innerSup)
         }
         function emitMultiLnStrLitExp(s: ts.NoSubstitutionTemplateLiteral | ts.TaggedTemplateExpression): ExpRes {
-            if (ts.isNoSubstitutionTemplateLiteral(s))
-                return asExpRes(`"""${s.text}"""`)
+            if (ts.isNoSubstitutionTemplateLiteral(s)) {
+                let lines = s.text.split('\n').map((l, i) => !!l.length ? indent1(l) : l);
+                return asExpRes(`"""${lines.join('\n')}"""`)
```

Comment:
likewise, can't indent1 now handle multiline without splitting before hand?

Reply:
this is a little bit of a hack but the split->indent->join actually skips the first and last lines, since the symbol text we generate has empty newlines before/after the literal, so we output something like

```
"""
    . . . 
    . 1 .
"""
```

where only the inside is indented..... now that i'm writing this out i think i should probably strip empty lines and just wrap the indented block in non-indented triple quotes. will update!